Library/Homebrew/tap.rb

```diff
@@ -1401,6 +1401,13 @@ def new_cask_path(token)
     cask_dir/cask_subdir/"#{token.downcase}.rb"
   end
 
+  sig { params(token: String).returns(Pathname) }
+  def new_font_path(token)
```

Comment:
```suggestion
  def new_cask_font_path(token)
```

also needs to have a `new_cask_font_path` definition or alias